vital/util/string.cxx

```diff
@@ -45,7 +45,8 @@ namespace vital {
 std::string
 string_format( const std::string fmt_str, ... )
 {
-  int final_n, n = ( (int)fmt_str.size() ) * 2; /* Reserve two times as much as the length of the fmt_str */
+  int final_n;
+  int n = static_cast<int>(fmt_str.size()) * 2; /* Reserve two times as much as the length of the fmt_str */
```

Comment:
Ack, you got rid of the spaces here, where elsewhere is adding them :slightly_smiling_face:.